### DISCUSSION TITLE
Sub menu three levels depth

### DISCUSSION THREAD

**Discussion by jonwebprocoder** [bootscore/bootscore#267] [Q&A] [answered]

**Comment by jonwebprocoder** [comment]
Hi

I am working on a development site and the sub -> sub menu items are not showing


The menu "who we are" should have 2 more levels of items as per the menu i have set-up in the screenshot

<img width="425" alt="image" src="https://user-images.githubusercontent.com/31352149/192815809-0c966c48-27fa-45a3-812a-95265ff2b2b9.png">

I am using this code

 <div class="collapse navbar-collapse" id="navcol-1">
          <ul class="navbar-nav ms-auto">


            <?php
            wp_nav_menu(array(
              'theme_location' => 'main-menu',
              'container' => false,
              'menu_class' => '',
              'fallback_cb' => '__return_false',
              'items_wrap' => '<ul id="%1$s" class="navbar-nav me-auto mb-2 mb-md-0 %2$s">%3$s</ul>',
              'depth' => 3,
              'walker' => new bootstrap_5_wp_nav_menu_walker()
            ));
            ?>
        </div>

Am i doing something wrong or is it a bug?

**Comment by justinkruit** [comment]
By default Bootstrap does not support submenus inside submenus. Maybe @crftwrk has some alternative code to get it to work.

**Comment by crftwrk** [comment]
As @justinkruit wrote, Bootstrap does not support a third level and nav-walker doesn't support it too.

Check this issue https://github.com/AlexWebLab/bootstrap-5-wordpress-navbar-walker/issues/19. 

@imanishpushkar modified the nav-walker https://github.com/AlexWebLab/bootstrap-5-wordpress-navbar-walker/issues/19#issuecomment-1234198658 to three level depth.

Register the nav-walker is a pluggable function. Simply add new nav-walker to your child and override function.

```php
function register_navwalker() {
  require_once('inc/class-bootstrap-5-navwalker.php'); // Change new path and file name
  // Register Menus
  register_nav_menu('main-menu', 'Main menu');
  register_nav_menu('footer-menu', 'Footer menu');
}
```

Does this help?

**Comment by jonwebprocoder** [comment, marked as answer]
Thanks i have a solution now here https://github.com/imanishpushkar/bs5-navwalker
